### PR TITLE
Add antd-open-browser extension

### PR DESCRIPTION
**Pull request by crazyair**
## Description

Open the [antd](https://ant-design.antgroup.com/components/overview-cn/) component

## Screencast

Metadata-7

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/antd-open-browser/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# antd-open-browser
+
+快速打开 antd 组件
```
Comment (pernielsentikaer) [inline extensions/antd-open-browser/README.md:3]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/antd-open-browser/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "antd-open-browser",
+  "title": "antd-open-browser",
```
Comment (pernielsentikaer) [inline extensions/antd-open-browser/package.json:4]:
```suggestion
  "title": "Antd",
```

File: extensions/antd-open-browser/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "antd-open-browser",
+  "icon": "logo.png",
+  "author": "crazyair",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "antd",
```
Comment (pernielsentikaer) [inline extensions/antd-open-browser/package.json:12]:
```suggestion
      "title": "Browse Antd",
```

File: extensions/antd-open-browser/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      "subtitle": "快速打开 antd 组件",
+      "description": "快速打开 antd 组件",
```
Comment (pernielsentikaer) [inline extensions/antd-open-browser/package.json:14]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

I left some suggestions for you to work with 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/antd-open-browser/package.json (hunk 4 above)
Comment (crazyair) [reply extensions/antd-open-browser/package.json]: fix

**Comment by crazyair** [comment]
thanks

**Comment by crazyair** [comment]
@pernielsentikaer merge？

**Comment by crazyair** [comment]
help

**Comment by crazyair** [comment]
@YodaLightsabr

**Review by pernielsentikaer — APPROVED** [review]

**Comment by crazyair** [comment]
image
How to optimize this? svg

**Comment by YodaLightsabr** [comment]
> @YodaLightsabr 

Am I needed here?

**Pull request merged**